examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -51,9 +52,14 @@ async function start(): Promise<void> {
     // Get or create the document depending if we are running through the create new flow
 
     const client = useFrs ? FrsClient :  new TinyliciousClient();
-    const clientResources = createNew
-        ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
-        : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
+
+    let clientResources;
+    if (createNew) {
```

Comment:
this demo does not need to stay like this. it's just an example of the detached flow working for demo purposes.